app/src/androidTest/java/com/github/sdp_begreen/begreen/fragments/SendPostFragmentTest.kt

```diff
@@ -73,8 +75,7 @@ class SendPostFragmentTest {
                     .thenReturn(user.id)
                 `when`(db.getAllUsers()).thenReturn(users)
 
```

Comment:
Because when you launch `CameraContainer`, it launches `CameraWithUIFragment` you have to mock this method.

It is needed if you apply the proposed modification as is.

```suggestion
`when`(db.getFollowedIds(any(), any())).thenReturn(emptyList())
```